src/accountsdb/fuzz.zig

```diff
@@ -303,7 +332,7 @@ pub fn run(seed: u64, args: *std.process.ArgIterator) !void {
                 maybe_incremental_file_info,
             );
 
-            const combined_manifest = try sig.accounts_db.FullAndIncrementalManifest.fromFiles(
+            var combined_manifest = try sig.accounts_db.FullAndIncrementalManifest.fromFiles(
                 allocator,
```

Comment:
Was this intentional?

Reply:
No, I must have been in a fugue state when this happened.

Reply:
249c66fcc56e3899e04d69dd42cc745f43be0f56